### PR TITLE
Promote Tasks API to GA

### PR DESCRIPTION
**Pull request by avijityadav**

### CODE REVIEW

File: api-reference/v1.0/api/basetasklist-list-tasks.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+ms.prod: "outlook"
+doc_type: apiPageType
+---
+
+# List baseTasks
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Get the [baseTask](../resources/basetask.md) resources from the tasks navigation property of a specific [baseTaskList](../resources/basetasklist.md).
```
Comment (JarbasHorst) [inline api-reference/v1.0/api/basetasklist-list-tasks.md:15]: Please, apply bold formatting to tasks.

File: api-reference/v1.0/api/basetasklist-list-tasks.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  "blockType": "ignored"
+}
+-->
+``` http
+GET /me/tasks/lists/{baseTaskListId}/tasks
+GET /users/{userId|userPrincipalName}/tasks/lists/{baseTaskListId}/tasks
+```
+
+## Optional query parameters
+This method supports `$expand`, `$filter` and `$top`  [OData query parameters](/graph/query-parameters) to customize the response.
```
Comment (JarbasHorst) [inline api-reference/v1.0/api/basetasklist-list-tasks.md:38]: Please, add "the" after "supports" and a comma after filter.

File: api-reference/v1.0/api/basetasklist-post-tasks.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+|Content-Type|application/json. Required.|
+
+## Request body
+In the request body, supply a JSON representation of the [baseTask](../resources/basetask.md) object.
+
+You can specify the following properties when creating a **baseTask**.
+
+|Property|Type|Description|
+|:---|:---|:---|
+|textBody|String|The task body in text format that typically contains information about the task.|
```
Comment (JarbasHorst) [inline api-reference/v1.0/api/basetasklist-post-tasks.md:50]: For consistency and to improve scanning, these properties should be in alphabetical order. Please, alphabetize the list.

File: api-reference/v1.0/api/basetask-update.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+|Authorization|Bearer {token}. Required.|
+
+## Request body
+[!INCLUDE [table-intro](../../includes/update-property-table-intro.md)]
+
+
+|Property|Type|Description|
+|:---|:---|:---|
+|textBody|String|The task body in text format that typically contains information about the task.|
+|createdDateTime|DateTimeOffset|The date in the specified time zone that the task was finished.|
```
Comment (JarbasHorst) [inline api-reference/v1.0/api/basetask-update.md:52]: Should it be "was created"? For clarity, I recommend the following revision:

- The date in the specified time zone when the task was created.

Please, apply this change also here:

- api-reference/v1.0/api/basetasklist-post-tasks.md

**Comment by JarbasHorst** [comment]
Hello @avijityadav, I reviewed your PR and left some comments. Please, take a look when you will have time.

File: api-reference/v1.0/resources/basetasklist.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+* User created task list ([taskList](../resources/tasklist.md) resource) 
+
+This is an abstract type.
+
+## Methods
+The following method apply to any of the derived types of **baseTaskList** (**wellKnownTaskList**,**taskList**)
+
+|Method|Return type|Description|
+|:---|:---|:---|
+|[List baseTaskLists](../api/tasks-list-lists.md)|[baseTaskList](../resources/basetasklist.md) collection|Get a list of the [baseTaskList](../resources/basetasklist.md) objects and their properties.|
```
Comment (JarbasHorst) [inline api-reference/v1.0/resources/basetasklist.md:29]: For consistency, I recommend you to replace "List baseTaskLists" with "List lists".

File: api-reference/v1.0/toc.yml [hunk 6]
```diff
@@ -6988,58 +6988,136 @@ items:
...
         - name: Get
-          href: api/linkedresource-get.md
+          href: api/linkedresource_v2-get.md
         - name: Update
-          href: api/linkedresource-update.md
+          href: api/linkedresource_v2-update.md
         - name: Delete
-          href: api/linkedresource-delete.md
+          href: api/linkedresource_v2-delete.md
+      - name: To-do Tasks(old)
```
Comment (JarbasHorst) [inline api-reference/v1.0/toc.yml:7066]: For consistency, please replace "Tasks" with "tasks" and add an empty space after tasks. => To-do tasks (old)

**Comment by JarbasHorst** [comment]
Hi @avijityadav, please include an update to the changelog file to surface your changes in the Microsoft Graph Changelog.

**Comment by avijityadav** [comment]
Hi @JarbasHorst ,

Due to a very recent re-org in our team, the GA of this API has been blocked and this API will not go to GA. I see that you have done a very detailed review. Sincere apologies for the inconvenience.

**Pull request closed without merging**